index.php

```diff
@@ -132,15 +140,47 @@
             $url = sprintf('team/%s', $platform);
             break;
     }
+    return $verifyUrl($url) ? $routes[$url] : $routes['default'];
```

Comment:
You can skip the `$verifyUrl` and use the PHP 7 syntax as this:

`return $routes[$url] ?? $routes['default'];`